setup.py

```diff
@@ -1,5 +1,6 @@
 import os
 from setuptools import setup, find_packages
+from config import __version__
```

Comment:
Just do `import config`. Use `config.__version__` here. And open one more PR to use everything else from the config file that can be used here (as that's not in the scope of this PR).

Reply:
Ok, I'll open a new PR